react/src/components/atoms/forms/InputNumber/InputNumber.stories.js

```diff
@@ -14,8 +14,16 @@ storiesOf('atoms/forms', module)
     'InputNumber', (() => {
       const inputTextOptionsWithKnobs = Object.assign(...Object.entries(InputNumberOptions).map(([k, v]) => (
         { [k]: v() })));
+      const storyProps = {
+        style: (inputTextOptionsWithKnobs.inline) ? { width: '400px' } : { width: '200px' }
```

Comment:
Is this only for testing purposes? do you want to you remove it to fix the visual regression?

Reply:
This is only for the story, but I should update the backstop image.